front/app/containers/Admin/users/UsersGroup.tsx

```diff
@@ -235,11 +235,9 @@ export class UsersGroup extends React.PureComponent<
   }
 }
 
-const UsersGroupWithHoCs = withRouter(
```

Comment:
It's a bit beyond my understanding how this worked before since we already need the Router props in the outermost component (the export default).